src/dawn/Serialization/SIRSerializer.cpp

```diff
@@ -15,7 +15,7 @@
 #include "dawn/SIR/SIR.h"
 #include "dawn/SIR/AST.h"
 #include "dawn/SIR/ASTVisitor.h"
-#include "dawn/SIR/SIR.pb.h"
+#include "dawn/SIR/SIR/SIR.pb.h"
```

Comment:
What was the intention behind this naming structure? wouldn't `dawn/SIR/proto/SIR.pb.h` be the better path? `SIR/SIR` seems a little redundant

Reply:
yes, it is redundant. However this is impose by proto buffer, if we want to create a module out of SIR (with SIR and statements) instead of standalone files. I think it is described here
https://github.com/protocolbuffers/protobuf/issues/1491

Reply:
I see, got it